packages/react-devtools-shared/src/devtools/views/ErrorBoundary/ErrorBoundary.js

```diff
@@ -73,6 +74,12 @@ export default class ErrorBoundary extends Component<Props, State> {
   }
 
   componentDidCatch(error: any, {componentStack}: any) {
+    logEvent({
+      event_name: 'error',
+      message: error.message ?? 'Unknown error',
+      stack: error.stack ?? 'Unknown stack',
```

Comment:
nit: Maybe this should be null (or empty string) if there's no stack, so we aren't forced to infer a missing stack by a human-friendly display string?